wire-ios-sync-engine/Source/UserSession/ZMUserSession.swift

```diff
@@ -267,6 +267,8 @@ public final class ZMUserSession: NSObject {
     let lastEventIDRepository: LastEventIDRepositoryInterface
     let conversationEventProcessor: ConversationEventProcessor
 
+    var supportedProtocolsService: SupportedProtocolsService?
+
```

Comment:
I don't think we need to store it, we can initialize the service in demand, also solve optionality